projects/Rockchip/devices/RK3568/README.md

```diff
@@ -0,0 +1,26 @@
+# RK3568
+
+This is a work-in-progress SoC device for RK3568
+
+**Known Issues/Limitations**
+
+* uses rkbin ddr init blob instead of u-boot tpl
+  - u-boot does not have any ddr init code
+* uses rkbin atf blob
+  - open source atf patches can boot the board but cpuidle sleep freezes the system
+* press spi switch on odroid-m1 during boot to run ddr init and u-boot from sd-card/emmc
+* ram is limited to ~4gb with `mem=3838M` (memory range `0x00200000` - `0xefffffff`)
```

Comment:
why ram limited to 4GB?

Reply:
As mentioned on next line there is/was an issue importing decoded video buffers into display driver when video buffers is located at >4GB ram address.

Removing the custom rk drm gem buffer part of driver resolves this issue, but that caused issues for older SoCs with multiple vop.
https://github.com/Kwiboo/linux-rockchip/commits/rk3568-fix-iommu/

There has been some major changes to iommu subsystem since then so reworking such change may resolve those issues.

Current workaround to limit to 4GB ram is an easy way out and is plenty of space to decode 1080p content.

Reply:
thank you for the detail. 4G Ram limit is related to video decode. thus, other usage can still use full 8G, right?

looking for the patch being mainlined.

Reply:
Yes, importing video buffers decoded by hantro for display on screen was the only issue I have found related to more then 4GB ram